andela/ah-fulldeck
# #163385143 Refactor email templates

### What does this PR do?
- It refactors our HTML templates to have a base html file that other templates can inherit from

### Description of this PR?
- Create css static file that helps us to style our email templates
- Create a base html file that holds all the static part of our templates
- Create an email verification and password reset html pages that extend from the base html and override the dynamic part

### How should this be manually tested?
- Run the server using `python manage.py runserver`
- Try the following endpoints to receive an email and see the templates used
` http://127.0.0.1:8000/api/v1/users/password_reset/` pass the email to reset the password for.
`http://127.0.0.1:8000/api/v1/users/` pass the user object to register a new user in order receive an email for verification.

### What are the relevant PT stories

[163385143](https://www.pivotaltracker.com/story/show/163385143)

### Screenshots

## Email Verification
<img width="745" alt="screenshot 2019-01-22 at 10 49 50" src="https://user-images.githubusercontent.com/23031493/51527709-1a578c80-1e46-11e9-8316-7771698ebc39.png">

## Password Reset
<img width="744" alt="screenshot 2019-01-22 at 10 50 09" src="https://user-images.githubusercontent.com/23031493/51527752-322f1080-1e46-11e9-89c0-f8ece3a898b9.png">
